### ISSUE TITLE
[cosmetic] Kestra email format could use some cleanup

### ISSUE THREAD

**Issue by johnsturgeon** [kestra-io/kestra#3527]
### Describe the issue

I set up an email alert and there are a couple issues:
1. the image in the resulting email is a bit blurry and misshapen
2. the backticks on the task indicate that it wants to be rendered as 'code' but it isn't
<img width="635" alt="image" src="https://github.com/kestra-io/kestra/assets/9746310/1831d747-e722-4e89-9261-8c86d7d32a17">

### Environment

- Kestra Version: 0.16.0
- Operating System (OS/Docker/Kubernetes): Docker
- Java Version (if you don't run kestra in Docker):

**Comment by sampath24-ss** [comment]
Is anyone working on this issue

**Comment by Ben8t** [comment]
@sampath24-ss no, up to you to provide a PR. We'll assign to another developer otherwise in the following days
